tests/flytekit/unit/types/file/test_image.py

```diff
@@ -0,0 +1,29 @@
+import io
+
+import requests
+
+from flytekit import task, workflow
+import PIL.Image
+
+
+@task(disable_deck=False)
+def t1() -> PIL.Image.Image:
+    url = "https://miro.medium.com/v2/resize:fit:1400/1*0T9PjBnJB9H0Y4qrllkJtQ.png"
+    response = requests.get(url)
+    image_bytes = io.BytesIO(response.content)
```

Comment:
Can you copy this image locally? Ideally unit tests should touch the network only in the absolute necessary case.

Reply:
updated it